SEED-platform/seed
# 2702 refactor/sample data for better

#### Any background context you want to provide?
The homepage has a feature allowing users to "Auto-Populate Sample Data". This generates 12 sample properties, one of which allows a user to run a BSync Analysis. At this time there is no sample property that allows users to run a BETTER Analysis.

#### What's this PR do?
- Adds a sample building to be used in a BETTER analysis ('Campus Building 2')
- Adds corresponding meter data to 'PM Meter Data.xlsx'. This data was taken from existing JCC building meters and modified. The Electrical usage data has been modified by a random percentage between +/- 15%, while the natural gas usage data has been modified by a random percentage between -3% and 3%
- Modifies the auto-populate modal to have instructions for a sample BETTER analysis
- Modifies the auto-populate modal style to encourage users to scroll down as the instructions extend beyond the viewing height of most screens.

#### How should this be manually tested?
On the homepage, select 'Auto-Populate Sample Data'. There should now be additional information under the 'BETTER' category. If the continue button is grayed out, follow instructions to auto-populate the sample data. 
Click Continue.
Go to Inventory.
Select building with PM Property ID = `5766981` and choose to run a BETTER analysis. The results should show examples of `Electricity Consumption` and `Fossil Fuel Consumption


#### What are the relevant tickets?
#2702 

#### Screenshots (if appropriate)
Gradient prompts users to scroll downward.
<img width="949" alt="Screen Shot 2021-11-05 at 1 44 45 PM" src="https://user-images.githubusercontent.com/58446472/140576361-795d7b3e-e5ce-45d9-aaa1-cb98b673acfa.png">
<img width="926" alt="Screen Shot 2021-11-05 at 1 44 51 PM" src="https://user-images.githubusercontent.com/58446472/140576369-70f6f588-fd97-4b41-a45e-f59ae4213d33.png">



BETTER Analysis for sample Building 5766981
![Screen Shot 2021-11-05 at 2 30 07 PM](https://user-images.githubusercontent.com/58446472/140575516-0ed768a9-2f6d-4096-be11-99567dc03269.png)
![Screen Shot 2021-11-05 at 2 30 27 PM](https://user-images.githubusercontent.com/58446472/140575514-c0bbc7a9-9675-4c98-9a08-b06e54a3ccc8.png)
